app/interactors/users/checkins/import_checkins.rb

```diff
@@ -14,6 +15,10 @@ def call
 
     private
 
+    def creat_activity
+      device.create_activity :import, owner: device.user, parameters: { count: CSV.read(path).length }
```

Comment:
@dmitrytrager Ok think I understand the benefits, not sure how to implement though. Where would this live? How exactly do you create an "activity actor" as described?